xml/System.Windows.Forms/HighDpiMode.xml

```diff
@@ -114,7 +114,7 @@
       </ReturnValue>
       <MemberValue>1</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The window queries for the DPI of the primary monitor once and uses this for the process on all monitors.</summary>
```

Comment:
application instead of process?

Reply:
@mairaw I think Ron suggested something similar too in your other PR.

Reply:
Sure I can change. This came from the /// comments.

```suggestion
        <summary>The window queries for the DPI of the primary monitor once and uses this for the application on all monitors.</summary>
```